src/tests/bulkConcepts/components/PreviewCard.test.js

```diff
@@ -48,6 +48,112 @@ describe('Test suite for ActionButton in BulkConceptsPage component', () => {
     expect(wrapper.length).toEqual(1);
   });
 
+  it('should render with mappings containing set members', () => {
+    const newProps = {
+      ...props,
+      concept: {
+        mappings: [
+          {
+            map_type: 'CONCEPT-SET',
+            to_concept_code: '1',
+            to_concept_name: 'LIVER FUNCTION TESTS',
+            source: '',
+          },
+        ],
+        descriptions: [{ description: '' }],
+      },
+    };
+    const wrapper = mount(<Router>
+      <PreviewCard {...newProps} />
+    </Router>);
+    expect(
+      wrapper.find('.synonyms').filterWhere(
+        node => node.text() === 'Set members',
+      ).length,
+    ).toEqual(1);
+  });
+
+  it('should render with mappings containing answers', () => {
+    const newProps = {
+      ...props,
+      concept: {
+        mappings: [
+          {
+            map_type: 'Q-AND-A',
+            to_concept_code: '1',
+            to_concept_name: 'LIVER FUNCTION TESTS',
+            source: '',
+          },
+        ],
+        descriptions: [{ description: '' }],
+      },
+    };
+    const wrapper = mount(<Router>
+      <PreviewCard {...newProps} />
+    </Router>);
+    expect(
+      wrapper.find('.synonyms').filterWhere(
+        node => node.text() === 'Answers',
+      ).length,
+    ).toEqual(1);
+  });
+
+  it('should call handleClick when more than five answers/set members are available', () => {
```

Comment:
Does this test mean that when more than five answers are available then handleClick is automatically called?

Reply:
Resolved. I have used a better description of the test.